src/components/Controls.vue

```diff
@@ -204,6 +204,12 @@ export default {
 		}
 	},
 
+	created: function() {
+		if (localStorage.getItem('nc_deck_compact_active') === 'true') {
```

Comment:
Let's use `deck.compactMode` as key here since that was also used in 0.8

```
js/controller/BoardController.js
186:		localStorage.setItem('deck.compactMode', JSON.stringify($rootScope.compactMode));
```